transmission-visualizer/build.gradle.kts

```diff
@@ -0,0 +1,69 @@
+import org.jetbrains.intellij.platform.gradle.TestFrameworkType
+
+plugins {
+    alias(libs.plugins.jetbrainsKotlinJvm)
+    id("org.jetbrains.intellij.platform") version "2.6.0"
+}
+
+group = "com.trendyol.transmission"
```

Comment:
These are hardcoded. We can get these from the properties

Reply:
Done